main.py

```diff
@@ -48,173 +47,14 @@ def create_pull_request_for_issue(*, token, owner, repo_name, issue, comments=No
             logger.info(f"Ignoring issue from non-allowed user: {issue['user']['login']}")
             return {"message": "Issue from non-allowed user ignored"}, 200
 
```

Comment:
@Aiderbot We shouldn't be deleting the main functions from the main.py. We are importing them from main.py in celery_tasks.py - please reinstate them.